sign-up.html

```diff
@@ -171,13 +171,7 @@
 
 
     </div>
-    <script>
-    const chk = document.getElementById('chk');
 
-chk.addEventListener('change', () => {
```

Comment:
@engineerscodes this changes aren't mentioned in the issue corresponding to this pr so please revert it.